src/main/java/com/google/error_analyzer/data/ErrorFixes.java

```diff
@@ -0,0 +1,71 @@
+/**Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+package com.google.error_analyzer.data;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.customsearch.Customsearch;
+import com.google.api.services.customsearch.CustomsearchRequestInitializer;
+import com.google.api.services.customsearch.Customsearch.Cse;
+import com.google.api.services.customsearch.Customsearch.Cse.List;
+import com.google.api.services.customsearch.model.Result;
+import com.google.api.services.customsearch.model.Search;
+import java.io.IOException;
+import java.lang.*;
+import java.security.GeneralSecurityException;
+import java.util.*;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+public class ErrorFixes{
+    
+    private static final Logger logger = LogManager.getLogger(ErrorFixes.class);
+    public String findFixes(String searchQuery) {
+    
+        try{
+            String searchEngine = "cx"; //Your search engine
```

Comment:
What is "cx"?
Should we create a constant?

Reply:
we need to replace cx with search engine id(can't put it on git)